apps/server/src/graphql/Task.ts

```diff
@@ -375,6 +388,58 @@ When the task is:
   })
 );
 
+builder.mutationField("addSubTask", (t) =>
```

Comment:
As I specified in the ticket, it should be `addSubtask`. I consider "subtask" as a single word not as 2 words ("sub task") nor conjoined words ("sub-task" -- it's also valid in the dictionary but not code friendly as it's got a dash).
```suggestion
builder.mutationField("addSubtask", (t) =>
```